src/temperature_component.cpp

```diff
@@ -441,8 +449,8 @@ unitval TemperatureComponent::getData( const std::string& varName,
             returnval = heatflux;
         } else if( varName == D_ECS ) {
             returnval = S;
-        } else if(varName == D_AERO_SCALE) {
-            returnval = alpha;
```

Comment:
why did you delete the alpha here?

Reply:
It was a duplicate.  Alpha is already processed on lines 434-435 (old numbering).

Reply:
ah, gotcha!